README.md

```diff
@@ -5,20 +5,32 @@
 
 ## Install
 
-Use the following command to instal the package as a dev dependency:
+Use the following command to install the package as a dev dependency:
 ```shell
-npm install @ottofeller/ofmt --save-dev
+npm install @ottofeller/ofmt --save-dev --save-exact
 ```
 
-The package implements an `install` command that copies `prettier` and `eslint` configs to your project - it creates a symlink to `prettier` config file and adds/extends `eslint` config within _package.json_. See the [Examples](#examples) sections for usage details.
-> 💡 The `install` script does not attempt to overwrite existing `prettier` config - if a _.prettierrc_ file exists it is left as is. An existing `eslint` config is extended with _eslint.quality.cjs_.
+Then the following command is used to install `prettier` and `eslint` configs as dev dependencies:
+```shell
+npx ofmt install
```

Comment:
@quesabe I probably already asked about this in the past, but shouldn't it be like this?
```sh
nix @ottofeller/ofmt install
```

Reply:
This way might be used when `ofmt` is not installed within a project.
```sh
npx @ottofeller/ofmt install
```

This one is legitimate as well. But it will search for a registered binary called `ofmt` (which is present if `ofmt` package is installed). Otherwise it will search registry for `ofmt` package, which does not exist leading to an error.
```sh
npx ofmt install
```

Ultimately the result is the same, but the behaviour differs in what `npx` is searching for: either a project or a bin file. The only pitfall for us is that the second option would trigger an error if the package is not installed before the command run.

Reply:
Got it, thanks! Let's keep the second option, I doubt someone will run the `install` script without prior installation of `ofmt`.